README.md

```diff
@@ -58,6 +106,31 @@ tenants:
           base_url: https://github.com
 ```
 
+## Configuration of the containers
+
+To configure the host serving the api and web UI, add
+`MONOCLE_HOST=<host or ip>` into the `.env` file. If you don't
+configure it, `MONOCLE_HOST` is set to `localhost`.
+
+If you want to use a specific version, add `MONOCLE_VERSION=<version>`
+into the `.env` file.
+
+### GitHub authentication
+
+If you want to protect the access to your indices, you can require a
+GitHub login to access and the people able to use the indeices will be
```

Comment:
Indices

Reply:
We should define which term we use: indice or index ?

Reply:
in fact indices and indexes are correct plural form of index according to https://grammarist.com/usage/indexes-indices/